src/Hazelcast.Net/Models/AuthenticationResult.cs

```diff
@@ -129,5 +132,8 @@ internal sealed class AuthenticationResult
         /// Cluster version. Read from KeyValuePairs["cluster.version"].
         /// </summary>
         public ClusterVersion ClusterVersion { get; }
+
+        /// <summary>
```

Comment:
Dangling doc?

Reply:
Yes, it is. Fixed at https://github.com/hazelcast/hazelcast-csharp-client/pull/902/commits/b246b6c0752b52869313042c36feb40935239ed6